corehq/ex-submodules/couchforms/openrosa_response.py

```diff
@@ -68,9 +69,9 @@ def response(self):
         return HttpResponse(self.xml(), status=self.status)
 
 
-SUCCESS_RESPONSE = get_openrosa_reponse(u'   √   ', ResponseNature.SUBMIT_SUCCESS, 201)
+SUCCESS_RESPONSE = get_openrosa_reponse(_(u'Form successfully saved'), ResponseNature.SUBMIT_SUCCESS, 201)
```

Comment:
are we allowed to change these responses? i think these also get returned from the [form submission API](https://confluence.dimagi.com/display/commcarepublic/Submission+API), so if someone was parsing for the previous message their workflow might break

Reply:
The API only guarantees that `message` is something human-readable, so clients shouldn't be depending on its exact contents, but, regardless, updated so that only formplayer submissions get the new message.